<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Farukh" failures="1" tests="3" name="Test" time="5.996" errors="0" timestamp="2024-12-02T19:29:10 EST">
  <testcase classname="class46.OrangeHRM" name="testLogo" time="0.043"/>
  <testcase classname="class46.OrangeHRM" name="testAppURL" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at class46.OrangeHRM.testAppURL(OrangeHRM.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppURL -->
  <testcase classname="class46.OrangeHRM" name="testHomePageTitle" time="0.001">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
